KevS-MULTISITES-REVAMPING-PROJECT

Kazarus4 opened the pull request:
One PR to govern all (fusion)

Kazarus4 committed:
7 to 30 screens done

Kazarus4 committed:
10 to 30 screens done

Kazarus4 committed:
11 to 30 screens done

Kazarus4 committed:
screens added

Kazarus4 committed:
Screens added

Kazarus4 committed:
Screens added

Kazarus4 committed:
Screensadded done

Kazarus4 committed:
Screen added

Kazarus4 committed:
Screens added done

Kazarus4 committed:
SK2465 #8673 FAQ WebHosting

Kazarus4 committed:
SK2464 #8665 Detach domain existing Website

Kazarus4 committed:
SK2461 #8661 Diagnostic Index of

Kazarus4 committed:
SK2460 #8658 3 guides -create sdmn -acces website view -config webhosting

Kazarus4 committed:
SK2459 #8657 Modify domain existing website

Kazarus4 committed:
SK2458 #8656 Htaccess rewrite mod

Kazarus4 committed:
SK2456 #8654 Website project

Kazarus4 committed:
SK2457 #8655 SQL change password

Kazarus4 committed:
SK2438 #8619 My website A AAAA diagnosis

Kazarus4 committed:
SK2429 #8618 My website add domain existing website

Kazarus4 committed:
SK2418 #8644 CDN using

Kazarus4 committed:
SK2419 #8645 Install 1 click module

Kazarus4 committed:
SK2420 #8647 Manage 1 click module

Kazarus4 committed:
SK2421 #8648 Diag errors 1 click module

Kazarus4 committed:
SK2422 #8643 Git integration WebHosting

Kazarus4 committed:
SK2424 #8610 Application Firewall

Kazarus4 committed:
SK2425 #8765 Multisites config multisites

Kazarus4 committed:
SK2426 #8612 Geolocation

Kazarus4 committed:
SK2428 #8623 Multisites website not installed

Kazarus4 committed:
SK2427 #8652 Multisites modify domain

Kazarus4 committed:
Merge pull request #8793 from ovh/KevS-REVAMP-MULTISITE-SCREEN-INTEGRATION

KevS-REVAMP-MULTISITE-SCREEN-INTEGRATION

Kazarus4 committed:
Resolution of conflicts

Kazarus4 committed:
SK2513 Exporter son site web FRFR version Done

Y0Coss committed:
SK2513 Exporter son site web FRFR - Proofreading FR

Kazarus4 committed:
SK2514 First steps web hosting FRFR version Done

Kazarus4 committed:
Merge branch 'KevS-MULTISITES-REVAMPING-PROJECT' of github.com:ovh/docs into KevS-MULTISITES-REVAMPING-PROJECT

Kazarus4 committed:
Update guide.fr-fr.md

Kazarus4 committed:
Merge branch 'KevS-MULTISITES-REVAMPING-PROJECT' of github.com:ovh/docs into KevS-MULTISITES-REVAMPING-PROJECT

Kazarus4 committed:
Update guide.fr-fr.md

Kazarus4 committed:
SK2514 First steps web hosting FRFR version Done

Kazarus4 committed:
Merge branch 'KevS-MULTISITES-REVAMPING-PROJECT' of github.com:ovh/docs into KevS-MULTISITES-REVAMPING-PROJECT

Kazarus4 committed:
SK2515 Put website online FRFR version Done

Y0Coss commented on pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.fr-fr.md:
```diff
@@ -102,11 +102,17 @@ Une fois tous les éléments en votre possession, la connexion à l'espace de st
+> Si, depuis votre [espace client OVHcloud](/links/manager), vous n'avez pas encore associé votre nom de domaine à un site web présent sur votre hébergement web, consultez [ce guide](/pages/web_cloud/web_hosting/my_websites_add_domain_existing_website).
 
-Pour vérifier le dossier dans lequel doit être publié le site web, positionnez-vous sur l'onglet `Mes sites`{.action} depuis votre espace client OVHcloud. Dans le tableau qui s'affiche, pour le site web souhaité, regardez le `Dossier racine`{.action} qui s'affiche. Publiez alors les fichiers du site dans ce dernier.
+Une fois connecté à votre espace de stockage, il ne vous reste plus qu'à mettre en ligne les fichiers de votre site. **Nous vous invitons à être particulièrement attentif quant au répertoire sur lequel vous allez télécharger les fichiers**. Ceci notamment si vous avez déclaré plusieurs sites web sur votre hébergement web.
```
```suggestion
Une fois connecté à votre espace de stockage, il ne vous reste plus qu'à mettre en ligne les fichiers de votre site. **Nous vous invitons à être particulièrement attentif quant au répertoire sur lequel vous allez télécharger les fichiers**, notamment si vous avez déclaré plusieurs sites web sur votre hébergement web.
```

Y0Coss commented on pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.fr-fr.md:
```diff
@@ -102,11 +102,17 @@ Une fois tous les éléments en votre possession, la connexion à l'espace de st
-Il se peut que vous trouviez sur votre espace de stockage un fichier s'intitulant « index.html ». Ce dernier peut avoir été créé par OVHcloud lors de l'installation de votre hébergement pour afficher une page par défaut sur votre site web. Si tel est le cas, n'oubliez pas de le supprimer lors de la mise en ligne de vos fichiers.
+Pour vérifier le dossier dans lequel doit être publié le site web, positionnez-vous sur l'onglet `Mes sites`{.action} depuis votre espace client OVHcloud. Dans le tableau qui s'affiche, pour le site web souhaité, regardez le `Dossier racine`{.action} qui s'affiche. Publiez alors les fichiers du site web dans ce dernier.
+
+Il se peut que vous trouviez sur votre espace de stockage un fichier s'intitulant « index.html ». Ce dernier peut avoir été créé par OVHcloud lors de l'installation de votre hébergement web pour afficher une page par défaut sur votre site web. Si tel est le cas, n'oubliez pas de le supprimer lors de la mise en ligne de vos fichiers.
```
```suggestion
Il se peut que vous trouviez sur votre espace de stockage un fichier intitulé « index.html ». Ce dernier peut avoir été créé par OVHcloud lors de l'installation de votre hébergement web pour afficher une page par défaut sur votre site web. Si tel est le cas, n'oubliez pas de le supprimer lors de la mise en ligne de vos fichiers.
```

Y0Coss commented on pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.fr-fr.md:
```diff
@@ -119,15 +125,14 @@ Il se peut que vous trouviez sur votre espace de stockage un fichier s'intitulan
->
 
-Aujourd'hui, la plupart des systèmes de gestion de contenu (CMS), tels que WordPress et Joomla! utilisent une base de données pour y stocker des éléments dits dynamiques, comme des commentaires ou des articles. Une connexion entre les fichiers du site et la base de données est donc indispensable afin que le site web puisse fonctionner correctement. Pour cela, il existe un fichier de configuration ...
+Aujourd'hui, la plupart des systèmes de gestion de contenu (CMS), tels que WordPress et Joomla! utilisent une base de données pour y stocker des éléments dits dynamiques, comme des commentaires ou des articles. Une connexion entre les fichiers du site web et la base de données est donc essential afin que le site web puisse fonctionner correctement. Pour cela, il existe un fichier de configuration ...
```
```suggestion
Aujourd'hui, la plupart des systèmes de gestion de contenu (CMS), tels que WordPress et Joomla!, utilisent une base de données pour y stocker des éléments dits dynamiques, comme des commentaires ou des articles. Une connexion entre les fichiers du site web et la base de données est donc essentielle afin que le site web puisse fonctionner correctement. Pour cela, il existe un fichier de configuration disposant des informations de la base qui permet cette connexion.
```

Kazarus4 committed:
Update pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.fr-fr.md

Co-authored-by: Yoann Cosse <63302382+Y0Coss@users.noreply.github.com>

Kazarus4 committed:
Update pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.fr-fr.md

Co-authored-by: Yoann Cosse <63302382+Y0Coss@users.noreply.github.com>

Kazarus4 committed:
Update pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.fr-fr.md

Co-authored-by: Yoann Cosse <63302382+Y0Coss@users.noreply.github.com>

Kazarus4 committed:
SK2513 - SK2514 - SK2515 - translations Done

Y0Coss commented on pages/web_cloud/web_hosting/exporter-son-site-web/guide.en-gb.md:
```diff
@@ -69,11 +69,36 @@ Once you have everything you need, there are two different ways of retrieving yo
+>>
+>> ![export-website](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/root-folders.png){.thumbnail}
+
+Still connected to your storage space, you just need to download the files of your website by accessing the previously identified root folder.
```
```suggestion
While still connected to your storage space, you just need to download the files of your website by accessing the previously identified root folder.
```

Y0Coss commented on pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.en-asia.md:
```diff
@@ -159,11 +164,11 @@ Once you have uploaded your files to your storage space and connected your datab
 If you observe that it does not display properly, we recommend:
 
-- **Checking the configuration of the domain name.** Your domain name’s DNS configuration may be preventing the website you have just uploaded to your OVHcloud Web Hosting plan from displaying correctly. Make sure that the A record currently configured in the DNS zone for your domain corresponds to the IP address of your OVHcloud Web Hosting plan.
+- **check the domain name configuration**: it is possible that the DNS configuration of the domain name does not allow it to display the website you have just uploaded to your OVHcloud web hosting. Make sure that the A record currently set in the DNS zone of your domain name corresponds to the IP address of your OVHcloud web hosting.
```
```suggestion
- **Checking the domain name configuration**: it is possible that the DNS configuration of the domain name does not allow it to display the website you have just uploaded to your OVHcloud web hosting. Make sure that the A record currently set in the DNS zone of your domain name corresponds to the IP address of your OVHcloud web hosting.
```

Y0Coss commented on pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.en-asia.md:
```diff
@@ -159,11 +164,11 @@ Once you have uploaded your files to your storage space and connected your datab
+- **check the domain name configuration**: it is possible that the DNS configuration of the domain name does not allow it to display the website you have just uploaded to your OVHcloud web hosting. Make sure that the A record currently set in the DNS zone of your domain name corresponds to the IP address of your OVHcloud web hosting.
 
-- **Making sure that no files are missing.** When uploading your files to the OVHcloud Web Hosting plan, you may have forgotten to transfer some files, or an error may have occurred. Be careful with this process, however, to avoid breaking any links between the website’s files and the database (if the website uses one).
+- **ensure that no files are missing**: it is possible that during the upload of the files to your OVHcloud web hosting, you forgot some files or an error occurred. Be careful during your operations to avoid breaking the link between the website files and the database (if it uses one).
```
```suggestion
- **Ensuring that no files are missing**: it is possible that during the upload of the files to your OVHcloud web hosting, you forgot some files or that an error occurred. Be careful during your operations to avoid breaking the link between the website files and the database (if it uses one).
```

Y0Coss commented on pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.en-asia.md:
```diff
@@ -159,11 +164,11 @@ Once you have uploaded your files to your storage space and connected your datab
+- **ensure that no files are missing**: it is possible that during the upload of the files to your OVHcloud web hosting, you forgot some files or an error occurred. Be careful during your operations to avoid breaking the link between the website files and the database (if it uses one).
 
-- **Checking that there are no errors in the website code.** This is definitely the most technical check to carry out, but the files you have uploaded may contain errors, and those errors may prevent the server from displaying your website correctly, or at all.
+- **check that the website code does not contain errors**: this check is probably the most technical, but it is possible that the files you have uploaded contain errors and do not allow the server to display your website correctly, or not at all.
```
```suggestion
- **Checking that the website code does not contain errors**: this check is probably the most technical, but it is possible that the files you have uploaded contain errors and do not allow the server to display your website correctly, or not at all.
```

Kazarus4 committed:
Update pages/web_cloud/web_hosting/exporter-son-site-web/guide.en-gb.md

Co-authored-by: Yoann Cosse <63302382+Y0Coss@users.noreply.github.com>

Kazarus4 committed:
Update pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.en-asia.md

Co-authored-by: Yoann Cosse <63302382+Y0Coss@users.noreply.github.com>

Kazarus4 committed:
Update pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.en-asia.md

Co-authored-by: Yoann Cosse <63302382+Y0Coss@users.noreply.github.com>

Kazarus4 committed:
Update pages/web_cloud/web_hosting/hosting_how_to_get_my_website_online/guide.en-asia.md

Co-authored-by: Yoann Cosse <63302382+Y0Coss@users.noreply.github.com>

Kazarus4 committed:
En duplications Done

benchbzh committed:
proofreading on 3 last translations

benchbzh committed:
minor update

Kazarus4 committed:
DE done

Kazarus4 committed:
Translations done

Kazarus4 committed:
Deletion of 1 sentence outdated done

benchbzh committed:
proofreading on translations ES-IT-PL-PT for add_domain_existing_website

Kazarus4 committed:
index of FR done

Kazarus4 committed:
MULTISITES CONFIGURE MULTISITE FR done

Kazarus4 committed:
MULTISITES MODIFY DOMAIN FR done

Kazarus4 committed:
My websites modify root folder FR doing

Kazarus4 committed:
Proof FR for index of multisite config multisite and multisite modify ndd done

Kazarus4 committed:
All trads for index of done

Kazarus4 committed:
All trads for multisite modify domain done

Kazarus4 committed:
MULTISITES CONFIGURE multisite all done

benchbzh committed:
root folder guide quick proof

benchbzh commented:
Proof done on the 3 last commits (see my last commit - sorry for this commit's name).

Kazarus4 committed:
Merge branch 'develop' into KevS-MULTISITES-REVAMPING-PROJECT

BrenKt committed:
update-fr

Y0Coss committed:
Phishing detection guide - Updating how to report them + Additional tips (#9133)

* Phishing detection guide - Updating how to report them to OVHcloud

* Phishing detection guide - Merging Outlook steps

* Proofreading

* Adding additional tips + translation & duplication

BrenKt committed:
update-fr_v1

thild42 committed:
Merge Kafka Connect and Kafka MirrorMaker into the Kafka category (#9127)

* Merge Kafka Connect and Kafka MirrorMaker into the Kafka category

* Updating reference categories of moved guides

---------

Co-authored-by: Yoann Cosse <yoann.cosse@ovhcloud.com>

Kazarus4 committed:
typo fixes done (#9139)

thild42 committed:
[Public Cloud Databases] Update Terraform guides (#9128)

* Update Terraform guide for db

* Update Terraform guide for analytics

* Update Terraform guide for analytics (this time)

* Fix user for analytics

* Proogfreading

* Date update before production

* Date update before production

---------

Co-authored-by: Yoann Cosse <yoann.cosse@ovhcloud.com>

OvhValentin committed:
feat(compute): update guide about resize instance (#9129)

* feat(compute): update guide about resize instance

* Proofreading - Translation - Images deletion

---------

Co-authored-by: Yoann Cosse <yoann.cosse@ovhcloud.com>

ThibautOVH committed:
FIX: Remove broken link (#9137)

* FIX: Remove broken link

Removed the link to the OVHcloud OPCP documentation (broken)

* Remove broken link

Remove broken link

* Update guide with new date and additional resources

Updated the document with a new update date and added additional resources for training and support.

* Update the last updated date in the guide

* Fix only - Reverting date update

---------

Co-authored-by: Yoann Cosse <yoann.cosse@ovhcloud.com>

dariakts committed:
EN/FR - Add - Trident CSI availability banner to EFS guides (#9134)

* FR/EN - Add - EFS supports Trident CSI banner

Signed-off-by: Daria Fortunato <10145007+dariakts@users.noreply.github.com>

* FR/EN - Add - EFS supports Trident CSI banner to EFS docs

Signed-off-by: Daria Fortunato <10145007+dariakts@users.noreply.github.com>

* Adding banner for all locales

---------

Signed-off-by: Daria Fortunato <10145007+dariakts@users.noreply.github.com>
Co-authored-by: Daria Fortunato <10145007+dariakts@users.noreply.github.com>
Co-authored-by: Yoann Cosse <yoann.cosse@ovhcloud.com>

Jessica41 committed:
Minor fixes on various guides (#9138)

* content review and fixes

* content review and fixes

BrenKt committed:
update_all

BrenKt committed:
translate_all

BrenKt committed:
update_v2

Y0Coss committed:
Control Panel access template - Account management guides (#9145)

Kazarus4 committed:
Optimisations CP NAV + Tabs preparations done for domains (#9140)

* Optimisations tabs cp nav prerequisistes done

* Optimisations tabs cp nav prerequisistes done

* Optimisations tabs cp nav prerequisistes done

* Optimisations tabs cp nav prerequisistes done

* Optimisations tabs cp nav prerequisistes done

BrenKt committed:
Merge pull request #9141 from ovh/Audit-WEB-Emails-alloffers-update-03-2026

[Audit] WEB - Emails - multiple update and fix EU/CA/US/APAC

BrenKt committed:
fix

BrenKt committed:
Merge pull request #9146 from ovh/Quickfix-WEB-Emails-fix_date-050326

[QuickFix] WEB - Emails - fixing dates email_generality + faq_email

BrenKt committed:
dates

BrenKt committed:
rounded-images

BrenKt committed:
rollback template email

BrenKt committed:
Merge pull request #9131 from ovh/SK-2410-WEB-Emails-Locked_for_spam

[SK-2410] -WEB- Emails - Manage an account locked for spam - improvement

Kazarus4 committed:
Optimisations tabs cp nav prerequisistes done (#9147)

Y0Coss committed:
Control Panel access template - Billing guides (#9149)

P13K committed:
VMware - Lifecycle Policy updates (#9143)

* Update guide.fr-fr.md

- Remove AMD hardware 
- Update End of Sales of Premier to 2026-03-31
- Update end of Growth Premier

* Duplication

---------

Co-authored-by: Yoann Cosse <yoann.cosse@ovhcloud.com>

ToutouneOvh committed:
Backup Agent - First configuration guide - fix address and screenshot (#9148)

* fix address and screenshot

* Proofreading - Improvements - Translations

---------

Co-authored-by: aruelle <antoine.ruelle@corp.ovh.com>
Co-authored-by: Yoann Cosse <yoann.cosse@ovhcloud.com>

Y0Coss committed:
Formats updates - Replacing deprecated !faq format on all concerned guides (#9150)

* Formats updates - Replacing deprecated !faq format for API

* Formats updates - Replacing deprecated !faq format on all other guides

* Cleaning empty columns in tables

Y0Coss committed:
Object Storage - Improving docs (replacing PR #9099) (#9151)

* improve with copilot suggestions

* improve getting started with S3

* improve copilot recos

* improve docs

* improve docs

* improvements by copilot

* improve with copilot

* improve with copilot

* improve winscp

* post-review small fixes

* Proofreading

* FR versions updates

* Date update & duplication

* Fixing date update

---------

Co-authored-by: lason-ovh <laurent.song@corp.ovh.com>

Hadramos committed:
feat(postgresql): Add PostgreSQL 18 (#9152)

Co-authored-by: Cédric Petit <cedric.petit@ovhcloud.com>

P13K committed:
VMware - Lifecycle Policy updates (#9153)

* Update guide.fr-fr.md

Add Granite lineup

* Proofreading & duplication

---------

Co-authored-by: Yoann Cosse <yoann.cosse@ovhcloud.com>

Kazarus4 committed:
Optimisation control panel (CP NAV) for Web Cloud Databases + CP NAV deletions when added by mistake (#9154)

* Optimisations done

* Optimisations done

BrenKt committed:
full_review_fr

BrenKt committed:
translate+duplicate

BrenKt committed:
Merge branches 'develop' and 'SK-2589-WEB-Emails-diagnotic_header' of github.com:ovh/docs into develop

Y0Coss committed:
Control Panel access template - Backup Agent (#9158)

Jessica41 committed:
Control Panel Access template - PCI - Compute (#9157)

* Control Panel Access template - PCI - Compute

* Fixing over-aggressive edition

---------

Co-authored-by: Yoann Cosse <yoann.cosse@ovhcloud.com>

ThibautOVH committed:
docs(opcp): add section to disable RAID configuration (#9125)

* docs(opcp): add section to disable RAID configuration

* fix updated date

* Proofreading

---------

Co-authored-by: Yoann Cosse <yoann.cosse@ovhcloud.com>

Kazarus4 committed:
Web Hosting/Domains - CP NAV multisite revamp guides + typo fixes (#9159)

* Optimisations done

* Optimisations done

Kazarus4 committed:
Merge branch 'develop' into KevS-MULTISITES-REVAMPING-PROJECT
